### ISSUE TITLE
custom model

### ISSUE THREAD

**Issue by Innocentius-alt** [snowzach/hassio-addons#29]
Dear Zach,

I am begging you to teach me where i should put a custom model files (tflite & txt). I can not find folder 'models' in my HaOS((
I tried to put them in "share" folder and write new path in configuration but no luck. 

`2023-09-17 20:50:23,204 - doods.doods - ERROR - Could not create detector tflite/default: Could not open 'share/coco_ssd_mobilenet_v1_1.0_quant.tflite'.`

**Comment by snowzach** [comment]
I don't recall how to do it.. I feel like I put it in /share/models... And then maybe you put /share/models/filename in the config for the mode file? I'm going off memory from like several years ago...

**Comment by snowzach** [comment]
Yes, put the model in /share/models... and then update the config model path to be /share/model/filename...

**Comment by Innocentius-alt** [comment]
Thank you!! Your advice works and it made me happy but not for long((.
I bumped another issue.
